src/app/dataset-view/additional-components/metadata-component/components/form-components/checkbox-field/checkbox-field.component.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { ComponentFixture, TestBed } from "@angular/core/testing";
+import { FormControl, FormGroup, ReactiveFormsModule } from "@angular/forms";
+
+import { CheckboxFieldComponent } from "./checkbox-field.component";
+import { TooltipIconComponent } from "src/app/dataset-block/metadata-block/components/tooltip-icon/tooltip-icon.component";
+import { NgbTooltipModule } from "@ng-bootstrap/ng-bootstrap";
+
+describe("CheckboxFieldComponent", () => {
+    let component: CheckboxFieldComponent;
+    let fixture: ComponentFixture<CheckboxFieldComponent>;
```

Comment:
Test state of the control with different input value - true or false

Reply:
Add test for state.